app/models/votable_field.rb

```diff
@@ -0,0 +1,18 @@
+class VotableField < ActiveRecord::Base
+  belongs_to :conference
+  validates :title, :votable_type, presence: true
+
+  # ratyrate does not allow criterias to have spaces in them
+  validate :no_spaces_in_title
```

Comment:
You commented on the code that ratyrate does not allow votable_fields to have spaces, so that's what's proper. No spaces.